docs/UserGuide.md

```diff
@@ -533,13 +552,12 @@ Format: `exit`
 
 Acceptable values for `n/NAME`:
 - Alphanumeric word with or without spaces and should not be blank.
-- Error message for invalid name: “Names cannot be blank and should not contain any special characters”.
+- Error message for invalid name: “Names should only contain alphanumeric characters and spaces, and it should not be blank”.
 
 Acceptable values for `p/PHONE`:
 - Numbers with no spaces or special characters.
 - At least 3 numbers.
-- Error message for invalid phone number: “Phone numbers should contain only numbers, with no spaces or special characters”.
-
+- Error message for invalid phone number: "Invalid command format!" followed by an instruction on the proper usage of the command you were executing.
```

Comment:
Not too sure why the error message for this was changed! 
<img width="655" alt="image" src="https://github.com/AY2324S1-CS2103T-F11-2/tp/assets/61283948/2c3533e3-c350-436e-8ce0-5e167c0ad962">

Reply:
Resolved!